compiler/rustc_session/src/options.rs

```diff
@@ -1345,8 +1345,10 @@ options! {
         "hash spans relative to their parent item for incr. comp. (default: no)"),
     incremental_verify_ich: bool = (false, parse_bool, [UNTRACKED],
         "verify incr. comp. hashes of green query instances (default: no)"),
+    inline_llvm: bool = (true, parse_bool, [TRACKED],
+        "enable LLVM inlining (default: yes)"),
     inline_mir: Option<bool> = (None, parse_opt_bool, [TRACKED],
-        "enable MIR inlining (default: no)"),
+        "enable MIR inlining (default: off)"),
```

Comment:
No other options use `default: off`, so I think this should stay as `default: no`.

Reply:
I don't know why `inline_mir` is an `Option<bool>` rather than just a `bool`. The comment on `parse_opt_bool` says:
```rust
    /// Use this for any boolean option that lacks a static default. (The
    /// actions taken when such an option is not specified will depend on
    /// other factors, such as other options, or target options.)
```
Seems like `inline_mir` has an obvious static value. Anyway, not really important for this PR.

Reply:
Sure, changed back. Thanks.

Reply:
My understanding is that MIR inlining is not on or off by default but depends on other options: whether it's a debug or release build, whether incremental is enabled.

Reply:
Ok, then the use of `Option<bool>` is good for `inline_mir`. Thanks for the clarification.